src/integrationTest/java/uk/gov/moj/sdt/base/WireMockBaseTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.moj.sdt.base;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.moj.sdt.cmc.consumers.client.BaseXmlTest;
+
+import java.nio.charset.Charset;
+import javax.inject.Inject;
+
+@ActiveProfiles("integration-test")
+@RunWith(SpringRunner.class)
```

Comment:
`@RunWith` appears to be JUnit4.  Should this use `@ExtendWith(SpringExtension.class)` instead?